release/release.go

```diff
@@ -108,3 +64,26 @@ func ReadLsb() (*Lsb, error) {
 
 	return lsb, nil
 }
+
+var dpkgStatusPath = "/var/lib/dpkg/status"
+
+// OnClassic returns whether the process is running inside a
+// classic Ubuntu system or a native Ubuntu Core image.
+func OnClassic() bool {
```

Comment:
Fixed, thanks. I considered caching, but decided it wasn't worth the trouble for now. We'll need to consider concurrency and will need to reset for tests, which is easy but more complex. If we end up using this in a relevant path that justifies the complexity, happy to fix it.